pangeo_forge_recipes/recipes/reference_hdf_zarr.py

```diff
@@ -76,6 +77,15 @@ class HDFReferenceRecipe(BaseRecipe, FilePatternRecipeMixin):
     xarray_open_kwargs: dict = field(default_factory=dict)
     xarray_concat_args: dict = field(default_factory=dict)
 
+    def _to_pipelines(self) -> ParallelPipelines:
+        """Translate recipe to pipeline for execution.
+        """
+        pipeline = []  # type: MultiStagePipeline
+        pipeline.append(Stage(self.store_chunk, list(self.iter_chunks())))
+        pipeline.append(Stage(self.finalize_target))
+        pipelines = [pipeline]  # type: ParallelPipelines
+        return pipelines
```

Comment:
Here note that the compilation of the flow to pipelines is now implemented by each recipe class, not BaseRecipe. This gives recipes a lot more flexibility.